### PR TITLE
docs: Release notes for 0.15.3

### PR DESCRIPTION
**Pull request by determined-dsw**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,17 @@
...
 
+Version 0.15.3
+==============
+
+**Release Date:** May 5, 2021
+
+**Bug Fixes**
+
+-  Images: Fix GPU support in CUDA 10.2 + Tensorflow 1.15 images.
+-  Trials: Fix trial environment dependency compatibility for
```
Comment (justin-determined-ai) [inline docs/release-notes.txt:19]: non-blocking: Seems a bit too implementation-detail focused (also afaik this wasn't our fault); maybe "Update to match `websockets` library API change." or something? @mackrorysd maybe I have this wrong

**Review by justin-determined-ai — APPROVED** [review]

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,17 @@
...
+   ``websockets>=9.0``.
```
Comment (stoksc) [inline docs/release-notes.txt:20]: I forgot to add one for my fix, maybe `Trials: Fix a bug that caused trials to panic upon receiving too many rendezvous addresses` or something?

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,17 @@
...
+
```
Comment (stoksc) [inline docs/release-notes.txt:21]: I forgot to add one for my fix, maybe `Trials: Fix a bug that caused trials to panic upon receiving too many rendezvous addresses` or something?
```suggestion
-  Trials: Fix trial environment dependency compatibility for
   ``websockets>=9.0``.
-  Trials: Trials: Fix a bug that caused trials to panic upon
    receiving too many rendezvous addresses.

```

File: docs/release-notes.txt (hunk 1 above)
Comment (determined-dsw) [reply docs/release-notes.txt:19]: @mackrorysd Any comment on this or should I just ship it?

Comment (ioga) [reply docs/release-notes.txt:19]: I wrote that line. I'm fine with either version as long as it mentions `websockets>=9.0`.

Comment (mackrorysd) [reply docs/release-notes.txt:19]: Sorry I must have missed the earlier tag. Maybe "Update to match websocket >= 9.0 library API change" to be specific and emphasize it's an incompatibility in a new version of a dependency.

**Review by mackrorysd — APPROVED** [review]

**Pull request merged**